### DISCUSSION TITLE
Why does Design View not remember the cursor position when zooming in?

### DISCUSSION THREAD

**Discussion by Daniil-Kolbasov** [AvaloniaUI/AvaloniaVS#420] [Q&A] [answered]
When you zoom in Design View on any area (for example, the top left), it is constantly reset to the center of the screen, although I use the cursor to indicate where I need to go. This is very annoying when you need to quickly move to some area with a large zoom

![ZoomInDesignView](https://github.com/AvaloniaUI/Avalonia/assets/34882813/32c59461-4038-400a-a0c5-27eb2d1a9896)

Microsoft Visual Studio Professional 2022 (64-bit) - Current
Version 17.7.7

**Comment by timunie** [comment]
@Takoooooo

**Reply by Daniil-Kolbasov** [reply]
What does it mean?

**Reply by timunie** [reply, marked as answer]
it means that tako is more experienced in this area than me. So I pinged him hoping he has some time to investigate. Probably something we want to implement for the previewer.
